src/Symfony/Component/Serializer/DependencyInjection/SerializerPass.php

```diff
@@ -67,8 +70,37 @@ public function process(ContainerBuilder $container): void
             }
         }
 
+        $denormalizers = [];
+        $trueNormalizers = [];
+        /** @var Reference|Definition $definition */
```

Comment:
According to `PriorityTaggedServiceTrait`, `$normalizers` is a list of `Reference`

Reply:
Yes. But we also add more items to this array. See around line 60. 

```php
foreach ($normalizers as $i => $normalizer) {
   $normalizers[$i] = $container->register('.debug.serializer.normalizer.'.$normalizer, TraceableNormalizer::class)
        ->setArguments([$normalizer, new Reference('serializer.data_collector')]);
}
```